test/testautomation_pixels.c

```diff
@@ -158,7 +154,7 @@ int pixels_allocFreeFormat(void *arg)
             SDLTest_AssertCheck(result->format == format, "Verify value of result.format; expected: %" SDL_PRIu32 ", got %" SDL_PRIu32, format, result->format);
             SDLTest_AssertCheck(result->BitsPerPixel > 0, "Verify value of result.BitsPerPixel; expected: >0, got %u", result->BitsPerPixel);
             SDLTest_AssertCheck(result->BytesPerPixel > 0, "Verify value of result.BytesPerPixel; expected: >0, got %u", result->BytesPerPixel);
-            if (result->palette != NULL) {
+            if (result->palette != NULL && ! SDL_ISPIXELFORMAT_FOURCC(result->format)) {
```

Comment:
Please remove the space between '!' and SDL

Reply:
done